src/core/symbology-ng/qgsgraduatedsymbolrendererv2.cpp

```diff
@@ -1348,6 +1315,52 @@ void QgsGraduatedSymbolRendererV2::setSourceColorRamp( QgsVectorColorRampV2* ram
   mSourceColorRamp.reset( ramp );
 }
 
+double QgsGraduatedSymbolRendererV2::minSymbolSize() const
+{
+  double min = 0;
+  for ( int i = 0; i < mRanges.count(); i++ )
+  {
+    double sz = 0;
+    if ( dynamic_cast< QgsMarkerSymbolV2 * >( mRanges[i].symbol() ) )
```

Comment:
this could be simplified to:

    if ( mRanges[i].symbol()->type() == QgsSymbolV2::Marker )

same for 1326, 1339, 1341